pkg/core/interop/context.go

```diff
@@ -119,7 +118,7 @@ func NewContractMD(name string) *ContractMD {
 	emit.Syscall(w.BinWriter, interopnames.SystemContractCallNative)
 
 	c.Script = w.Bytes()
-	c.Hash = hash.Hash160(c.Script)
+	c.Hash = state.CreateNativeContractHash(c.Script)
```

Comment:
`CreateNativeContractHash(name string)`, move all these binwriter things into the function.

Reply:
Fixed.